### PR TITLE
Fbx loader support targets for cameras and lights

### PR DESCRIPTION
**Pull request by looeee**
(note: Ambient light code is here too, so hopefully if that gets merged first there will be no conflicts 🤞 )

I was adding support for camera targets, but noticed that there is actually a generic 'LookAtProperty' for all model types, so I've mapped this as follows:

1. If the object has a `.target` property, set the `target.position` and add the `target` to the scene graph (lights)

2. If there is no `target`, use Object3D.lookAt() to point the object towards the target (cameras and other Object3Ds)

Also, if the target is at (0, 0, 0), do nothing.

### CODE REVIEW

**Comment by looeee** [comment]
Notes for `Object3D.lookAt` say:

> This method does not support objects with rotated and/or translated parent(s)

Does this mean that it just won't do anything in this case, or that it will work incorrectly? 

If it's the latter we might need to put in some additional checks here, or set this to only work for cameras and lights.

**Comment by looeee** [comment]
Oh, and one other thing - I'm using `Array.Prototype.map` here. 

Is that OK?

**Comment by Mugen87** [comment]
I would say yes.

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/loaders/FBXLoader.js [hunk 1]
```diff
@@ -1755,6 +1751,47 @@
...
+							var pos = lookAtTarget.properties.Lcl_Translation.value.split( ',' ).map( function ( val ) {
+
+								return parseFloat( val );
+
+							} );
+
+							// DirectionalLight, SpotLight
+							if ( model.target !== undefined ) {
+
+								model.target.position.set( pos[ 0 ], pos[ 1 ], pos[ 2 ] );
```
Comment (mrdoob) [inline examples/js/loaders/FBXLoader.js]: Btw, for these cases you can just do...

```js
model.target.position.fromArray( pos );
```

Comment (looeee) [reply examples/js/loaders/FBXLoader.js]: Ah, I forgot! Thanks :)

**Pull request merged**